src/trading/IClaimProceeds.sol

```diff
@@ -0,0 +1,6 @@
+pragma solidity ^0.4.13;
+
+
+contract IClaimProceeds {
+
```

Comment:
Why does this not have any functions declared?

Reply:
No other contracts call anything on it.  This is basically just a placeholder in order to keep the pattern consistently followed.  I suspect at some point we'll have contracts that call this, at which point the interface will be filled out.